erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
```

Comment:
As methods in erizo are named "muteStream" I guess we are going to include the possibility of muting video in the future. Why we don't use this nomenclature also in the client (stream.mute(audioActive, videoActive))?